content/chainguard/chainguard-images/recommended-practices/updating-images/strategies-tools-updating-images/index.md

```diff
@@ -0,0 +1,141 @@
+
+
+## Not Updating Means More Risk
+
+End of Life (EOL) software [presents a host of security risks](https://www.chainguard.dev/unchained/why-end-of-life-software-means-400-cves-per-year). Upgrading may require a great deal of work and proper testing to be in place, but it's a small price to pay to keep your systems from being at risk and accruing technical debt.
+
+If your application has an automated test suite with good coverage, you can be confident that any breakages caused by upgrades will be caught before deployment to production. Of course, whenever a test failure occurs, there will be work required to address it. Putting off that work by delaying upgrades will only mean that more work is required in the future as further changes pile up.
+
+Another way organizations test and reduce the risk of breaking changes introduced by updates is through the use of staging environments where changes are tried out before being pushed to production. An alternative approach to this is "[*testing in production*](https://increment.com/testing/i-test-in-production/)" which typically involves using techniques like [*feature flags*](https://www.honeycomb.io/blog/what-is-a-feature-flag-best-practices-and-use-cases) and staged updates to verify the effects of changes before they impact the majority of users.
+
+
+## Knowing When Updates are Available
+
+The primary way of knowing when a new image is available is through the registry itself. Many registries will offer a webhook callback service ([Docker Hub](https://docs.docker.com/docker-hub/webhooks/), for example), but this is typically only for your own repositories. If you want to get notified when a public repository is updated, you'll typically have to use a third-party service like [*NewReleases*](https://newreleases.io/).
+
+If you're trying to find out how outdated the images in your Kubernetes cluster are, you might find the [*version-checker*](https://github.com/jetstack/version-checker) project to be useful. This is a Kubernetes utility that will create an inventory of your current images and produce a chart showing how out of date they are. The dashboard can form part of a full solution with notifications for outdated software being sent to cluster administrators for mitigation.
+
+<center><img src="stay_secure_1.png" alt="Image of table titled Cluster Image Version Checks. This table is a grafana dashboard with four colums: Image, Current Vresion, Latest Version, and Is Latest." style="width:950px;"></center>
```

Comment:
Ugh, thanks for the info i was unaware

I added "images not centering" to the agenda for today's meeting but we didn't get to it. I'm going to leave this for now because there are other problems with images (sizing in some cases) so I want this to at least look good on most browsers (for now, anyway), Eventually we can do a scan to automatically reconvert any HTML image code to plain markdown